crates/telio-proto/protos/upgrade.proto

```diff
@@ -9,6 +9,10 @@ enum Decision {
 message Upgrade {
 	string endpoint = 1;
 	fixed64 session = 2;
+	// endpoint type used by the sender
+	uint32 endpoint_type = 3;
+	// endpoint type selected for the receiver
+	uint32 receiver_endpoint_type = 4;
 }
```

Comment:
The pong message tells the upgrade initiator the type of the endpoint of the other side. Here we are telling the upgrade responder the type of our endpoint, and the type of theirs endpoint. You could argue that the second type is not strictly needed but I disagree. Since this is a decision made by the initiator, I think that the responder shouldn't guess from the endpoint its type (even if it is technically possible).